isolate_example/lib/page_one.dart

```diff
@@ -12,20 +12,115 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
-class PerformancePage extends StatelessWidget {
+class PerformancePage extends StatefulWidget {
+  @override
+  _PerformancePageState createState() => _PerformancePageState();
+}
+
+class _PerformancePageState extends State<PerformancePage> {
+  Future<void> computeFuture = Future.value();
+
   @override
   Widget build(BuildContext context) {
     return Center(
       child: Column(
         mainAxisSize: MainAxisSize.min,
         children: <Widget>[
           SmoothAnimationWidget(),
+          Container(
+            alignment: Alignment.bottomCenter,
+            padding: EdgeInsets.only(top: 150.0),
+            child: Column(
+              children: <Widget>[
+                FutureBuilder<void>(
+                  future: computeFuture,
+                  builder: (BuildContext context, AsyncSnapshot snapshot) {
+                    return RaisedButton(
+                      child: const Text('Compute on Main'),
+                      elevation: 8.0,
+                      onPressed: mainIsolateCallBack(context, snapshot),
+                    );
+                  },
+                ),
+                FutureBuilder<void>(
+                  future: computeFuture,
+                  builder: (BuildContext context, AsyncSnapshot snapshot) {
+                    return RaisedButton(
+                      child: const Text('Compute on Secondary'),
+                      elevation: 8.0,
+                      onPressed: secondaryIsolateCallBack(context, snapshot),
+                    );
+                  },
+                ),
+              ],
+            ),
+          ),
         ],
       ),
     );
   }
+
+  VoidCallback mainIsolateCallBack(
```

Comment:
nit: Give these a name that indicates they *create* the callback rather than *are* the callback. Something like `createMainIsolateCallback` or `getMainIsolateCallback`, for example.